prov/verbs/configure.m4

```diff
@@ -19,7 +19,30 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 				[],
 				[$verbs_PREFIX],
 				[$verbs_LIBDIR],
-				[verbs_ibverbs_happy=1],
+				[
+				echo -n "Checking if libibverbs is linkable by libtool... "
+				file=conftemp.$$.c
+				rm -f $file conftemp
+				cat > $file <<EOF
+/* Override any GCC internal prototype to avoid an error.
+   Use char because int might match the return type of a GCC
+   builtin and then its argument prototype would still apply.  */
+#ifdef __cplusplus
```

Comment:
I'm pretty sure that `--tag=CC` will guarantee that libtool will use the C compiler/linker, and not the C++ one.  So you don't necessarily need the C++/extern "C" stuff.